packages/amplify_datastore_plugin_interface/test/testData/Blog.dart

```diff
@@ -152,6 +155,10 @@ class Blog extends Model {
     modelSchemaDefinition.name = "Blog";
     modelSchemaDefinition.pluralName = "Blogs";
 
+    modelSchemaDefinition.indexes = [
```

Comment:
Is there a difference between an empty array and an array with the `id` field marked with `@primaryKey`? Wondering if we need to always include the primary key field or not.

Reply:
Thanks for reviewing :)
This is simply following GraphQL transformer + codegen convention of how to describe an index, e.g. `@primaryKey`

`@model` with normal `id` as the default primary key
```
{ name: 'key', arguments: { fields: [ 'id' ] } }
```

`@model` with `modelID` annotated by `@primaryKey`
```
{ name: 'key', arguments: { fields: [ 'modelID' ] } }
```

`@model` with `productId` annotated by `@primaryKey` + sortKeyFields
```
{
  name: 'key',
  arguments: { fields: [ 'productId', 'productName', 'warehouseId', 'region' ] }
}
```

And `@index`

`@model` has `postId` annotated with `@index` + sortKeyFields
```
{
  name: 'byPost',
  arguments: { fields: [ 'postId', 'content' ] }
}
```